opendcs-rest-api/build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
     id "opendcs-rest-api.deps-conventions"
     id "opendcs-rest-api.publishing-conventions"
     id "war"
+    id 'com.bmuschko.tomcat' version '2.7.0'
```

Comment:
Seemed to work just fine for now, it might fall over with Tomcat > 9. I do get an error message about the JSP compiler not have the secret flag set when I bump up to the latest version of 9, but it doesn't seem to stop anything for the REST API since it doesn't use JSP (would probably have issues with the web client).